app/src/main/java/org/cru/godtools/ui/dashboard/tools/ToolsCategoryFragment.kt

```diff
@@ -1,10 +1,87 @@
 package org.cru.godtools.ui.dashboard.tools
 
+import android.os.Bundle
+import androidx.fragment.app.viewModels
 import dagger.hilt.android.AndroidEntryPoint
+import javax.inject.Inject
+import org.ccci.gto.android.common.androidx.lifecycle.onDestroy
 import org.cru.godtools.R
+import org.cru.godtools.base.Settings
 import org.cru.godtools.base.ui.fragment.BaseFragment
 import org.cru.godtools.databinding.DashboardToolsCategoryFragmentBinding
+import org.cru.godtools.download.manager.GodToolsDownloadManager
+import org.cru.godtools.model.Tool
+import org.cru.godtools.model.Translation
+import org.cru.godtools.ui.tooldetails.startToolDetailsActivity
+import org.cru.godtools.ui.tools.ToolsAdapter
+import org.cru.godtools.ui.tools.ToolsAdapterCallbacks
+import org.cru.godtools.ui.tools.ToolsAdapterViewModel
 
 @AndroidEntryPoint
-class ToolsCategoryFragment :
-    BaseFragment<DashboardToolsCategoryFragmentBinding>(R.layout.dashboard_tools_category_fragment)
+class ToolsCategoryFragment() :
```

Comment:
I don't think you need the empty default constructor here. It got flagged by code scanning